src/style.css

```diff
@@ -1 +1,62 @@
-/* add styles here */
+
+@font-face {
+    font-family: 'Roboto';
+    src: url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap');
+  }
+  
+
+  body {
```

Comment:
Using tag names for styling, such as `body`, is against best practices. It is recommended to use classes for styling to enhance maintainability and scalability.